oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/spi/editor/package-info.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This package contains index creation and editing related classes and interfaces.
```

Comment:
"editing related" is not the right term I think... The word "edit" comes from the oak "Editor", which is the mechanism used to observe changes (like a "trigger" in a relational database).

I would write something like "Contains classes that deal with updating indexes, including how to build indexing backend documents (e.g. Lucene documents)".